src/Raven.Server/Documents/TimeSeries/TimeSeriesStorage.cs

```diff
@@ -2014,6 +2014,29 @@ private static void MarkSegmentAsPendingDeletion(DocumentsOperationContext conte
             }
         }
 
+        public long GetNumberOfTimeSeriesSegmentsToProcess(DocumentsOperationContext context, string collection, in long afterEtag, out long totalCount)
```

Comment:
Is it segments or the number of time series?

Reply:
Segments